superset/utils/log.py

```diff
@@ -359,3 +365,29 @@ def log(  # pylint: disable=too-many-arguments,too-many-locals
         except SQLAlchemyError as ex:
             logging.error("DBEventLogger failed to log event(s)")
             logging.exception(ex)
+
+
+class StdOutEventLogger(AbstractEventLogger):
```

Comment:
NOTE: side mission, creating this to use in my `superset_config` made it easier to test/debug. This `StdOutEventLogger` could be a decent default overall, or at least for when we're in development / debug mode. Though changing the default would be a breaking change so I'm holding back here for now.